doc/architecture/platforms/snp.rst

```diff
@@ -0,0 +1,28 @@
+AMD SEV-SNP
+===================
+
+.. warning:: SEV-SNP support is currently experimental and in active development.
+
+How to use the AMD SEV-SNP platform
+-----------------------------------
+CCF must run on an AMD CPU which supports SEV-SNP.
+
+To use SNP, ``virtual`` must be specified in ``COMPILE_TARGETS``, at compile time. If any other targets are specified, you may also need to specify the ``TEST_ENCLAVE=virtual`` runtime argument.
+
+Attestation
+-----------
+SNP attestation provide several fields needed to establish trust.
```

Comment:
We need to explain that this is the deployment on Azure confidential containers on ACI, with a link to https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-containers

Reply:
https://github.com/microsoft/CCF/pull/4330/commits/9101ab52764ae85e91357c8016afe432f924e49a